kview/src/main/java/dev/ikm/komet/kview/mvvm/view/genediting/GenEditingDetailsController.java

```diff
@@ -209,14 +214,35 @@ private void initialize() {
         // clear all semantic details.
         semanticDetailsVBox.getChildren().clear();
         EntityFacade semantic = genEditingViewModel.getPropertyValue(SEMANTIC);
+        Composer composer = new Composer("Semantic composer");
+        genEditingViewModel.setPropertyValue(COMPOSER, composer);
 
         // if the semantic is null, then we generate a default one
         if (semantic == null) {
             genEditingViewModel.setPropertyValue(MODE, CREATE);
             EntityFacade pattern = genEditingViewModel.getPropertyValue(PATTERN);
 
+            StampViewModel defaultStampViewModel = new StampViewModel();
+            defaultStampViewModel.setPropertyValue(STATUS, State.ACTIVE);
+            defaultStampViewModel.setPropertyValue(AUTHOR, TinkarTerm.USER);
+            defaultStampViewModel.setPropertyValue(MODULE, TinkarTerm.DEVELOPMENT_MODULE);
+            defaultStampViewModel.setPropertyValue(PATH, TinkarTerm.DEVELOPMENT_PATH);
```

Comment:
Can this be obtained by the current view models stamp view model? Assuming the user populated the fields in the upper right corner popup. If possible we could check if missing use the above defaults.